src/groups/connections.jl

```diff
@@ -0,0 +1,174 @@
+    group_log!(M.manifold, Y, pinvq)
+    return translate_diff!(M.manifold, Y, p, Identity(M.manifold, p), Y)
+end
+
+"""
+    vector_transport_to(M::CartanSchoutenMinusGroup, ::Identity, X, q, ::ParallelTransport)
+
+Transport tangent vector `X` identity on the group manifold with the
+[`CartanSchoutenMinus`](@ref) connection to point `q`. See [^Pennec2020] for details.
+
+[^Pennec2020]:
+    > X. Pennec and M. Lorenzi, “5 - Beyond Riemannian geometry: The affine connection
+    > setting for transformation groups,” in Riemannian Geometric Statistics in Medical Image
+    > Analysis, X. Pennec, S. Sommer, and T. Fletcher, Eds. Academic Press, 2020, pp. 169–229.
+    > doi: 10.1016/B978-0-12-814725-2.00012-1.
+"""
+vector_transport_to(M::CartanSchoutenMinusGroup, Y, ::Identity, X, q, ::ParallelTransport)
+
+function vector_transport_to!(M::CartanSchoutenMinusGroup, Y, p, X, q, ::ParallelTransport)
+    return inverse_translate_diff!(M.manifold, Y, q, p, X, LeftAction())
+end
+
+"""
+    vector_transport_to(M::CartanSchoutenPlusGroup, ::Identity, X, q, ::ParallelTransport)
+
+Transport tangent vector `X` at identity on the group manifold with the
+[`CartanSchoutenPlus`](@ref) connection to point `q`. See [^Pennec2020] for details.
+
+[^Pennec2020]:
+    > X. Pennec and M. Lorenzi, “5 - Beyond Riemannian geometry: The affine connection
+    > setting for transformation groups,” in Riemannian Geometric Statistics in Medical Image
+    > Analysis, X. Pennec, S. Sommer, and T. Fletcher, Eds. Academic Press, 2020, pp. 169–229.
+    > doi: 10.1016/B978-0-12-814725-2.00012-1.
+"""
+vector_transport_to(M::CartanSchoutenPlusGroup, Y, ::Identity, X, q, ::ParallelTransport)
+
+function vector_transport_to!(M::CartanSchoutenPlusGroup, Y, p, X, q, ::ParallelTransport)
+    return inverse_translate_diff!(M.manifold, Y, q, p, X, RightAction())
+end
+
+"""
+    vector_transport_direction(M::CartanSchoutenZeroGroup, ::Identity, X, d, ::ParallelTransport)
+
+Transport tangent vector `X` at identity on the group manifold with the
+[`CartanSchoutenZero`](@ref) connection in the direction `d`. See [^Pennec2020] for details.
+
+[^Pennec2020]:
+    > X. Pennec and M. Lorenzi, “5 - Beyond Riemannian geometry: The affine connection
+    > setting for transformation groups,” in Riemannian Geometric Statistics in Medical Image
+    > Analysis, X. Pennec, S. Sommer, and T. Fletcher, Eds. Academic Press, 2020, pp. 169–229.
+    > doi: 10.1016/B978-0-12-814725-2.00012-1.
+"""
+vector_transport_direction(
+    M::CartanSchoutenZeroGroup,
+    Y,
+    ::Identity,
+    X,
+    d,
+    ::ParallelTransport,
+)
+
+function vector_transport_direction!(
+    M::CartanSchoutenZeroGroup,
+    Y,
+    p::Identity,
+    X,
+    d,
+    ::ParallelTransport,
+)
+    dexp_half = group_exp(M.manifold, d / 2)
+    translate_diff!(M.manifold, Y, dexp_half, p, X, RightAction())
+    return translate_diff!(M.manifold, Y, dexp_half, p, Y, LeftAction())
+end
+
+function vector_transport_to!(
```

Comment:
In general it's only correct along one parameter subgroups, so I restricted it to elements of the Lie algebra (even though it's a bit more general).

I didn't document it because, for consistency here, I've only put documentation for `vector_transport_direction` methods and this method of `vector_transport_to` is fairly straightforward. Should I just document this one method or explain the design differently?